### ISSUE TITLE
Demo ReadMe Error 

### ISSUE THREAD

**Issue by mangoggul** [NVlabs/SegFormer#161]
Hi, FIrst of all, Thx for your code. 
i just want to fix some little error

python demo/image_demo.py demo/demo.png local_configs/segformer/B1/segformer.b1.512x512.ade.160k.py \
/path/to/checkpoint_file --device cuda:0 --palette cityscapes

this is the instruction that you wrote in Readme

As far as i know, i think "--palette cityscapes" need to be changed to "--palette ade"
Am i right?

**Comment by crispto** [comment]
yes, use cityscapes will cause an error in  mmseg/models/segmentors/base.py", line 246, in show_result
assert palette.shape[0] == len(self.CLASSES)
